autoload/picker.vim

```diff
@@ -79,6 +79,14 @@ function! s:ListHelpTagsCommand() abort
     return 'cut -f 1 ' . join(findfile('doc/tags', &runtimepath, -1))
 endfunction
 
+function! s:CloseWindowAndDeleteBuffer() abort
+    " Close the current window, deleting buffers that are no longer displayed.
+    let l:bufhidden = &bufhidden
+    set bufhidden=delete
+    close!
+    let &bufhidden = l:bufhidden
```

Comment:
If the buffer is deleted at line 108, why do we need to restore the old value of bufhidden? Does that restoration affect the deleted vim-picker buffer, or the current buffer that replaced the deleted vim-picker buffer?  Just curious.

Reply:
Thanks, good catch! `bufhidden` is local to the vim-picker buffer so we needn't store the old value and restore it: I'll amend the commit.

Reply:
If I remove line 87, the patch still resolves #42.  I have a concern that we might inadvertently modify the new current buffer's `bufhidden` property, but testing shows no affect on `set bufhidden?` either way (with line 87, or without line 87).  Appears my concern is moot.